systests/spring-boot/src/test/java/org/apache/cxf/systest/jaxrs/spring/boot/SpringJaxrsComponentScanningTest.java

```diff
@@ -51,8 +52,8 @@ public void testCxfComponentScan() {
         // The component scanner only looks for CXF's @Provider annotations, 
         // not JAX-RS Features/@Provider.
         assertThat(scanner.getFeatures())
-                .hasSize(2)
-                .hasOnlyElementsOfTypes(OpenApiFeature.class, JAXRSBeanValidationFeature.class);
+                .hasSize(3)
+                .hasOnlyElementsOfTypes(ObservationFeature.class, OpenApiFeature.class, OpenApiFeature.class, JAXRSBeanValidationFeature.class);
```

Comment:
Seems like `OpenApiFeature.class` is included twice:

```suggestion
                .hasOnlyElementsOfTypes(ObservationFeature.class, OpenApiFeature.class, JAXRSBeanValidationFeature.class);
```